builds/codex-v103-aeturnum-monorepo/packages/core/src/codex_core/chronomerkl.py

```
@@ -0,0 +1,9 @@
+import hashlib, time, json
+class ChronoMerkle:
+    def __init__(self): self.events=[]
+    def add(self, info:dict):
+        ts=time.time(); blob=json.dumps({"ts":ts,"info":info},separators=("," ,":" )).encode()
```

Ruff (0.14.3)

Multiple statements on one line (semicolon)

(E702)